### DISCUSSION TITLE
NVIDIA control panel appears to merge stable and lazer program profiles into one

### DISCUSSION THREAD

**Discussion by bdach** [ppy/osu#26137] [Q&A] [answered]
As reported on twitch chat. See following video:

https://github.com/ppy/osu/assets/20418176/be496ff9-459d-400a-9ef7-7d9016d68477

Stable is the one forcing threaded optimisation off, but selecting the lazer executable seems to be reusing that profile.

**Comment by peppy** [comment]
This sounds pretty dire, and may be a reason for users reporting performance issues with lazer 😱

**Reply by GhostylGit** [reply]
I moved both osu!lazer and stable over to my old sata ssd from my hdd as I got a new m.2 and noticed a night a day difference in stuttering, it was almost completely gone. It would stutter every time I entered or exited skinning and if I spammed arrow keys in song select or sometimes when I loaded a map. I also noticed my draw fps jump from around 850 avg to 950-1000 during gameplay. I did still have a much smaller audio stutter, where my audio thread would jump up to around 20k, much less often than I used to but changing the threaded optimization to on seems to have fixed that as well.

I moved stable over to my ssd a couple of years ago but it had no effect on performance so I just left it on my hdd for storage space, so I didn't expect lazer's performance to be largely impacted.

**Reply by peppy** [reply]
SSD should have no perceivable difference, so it must be something else. Also, you're very much in the wrong discussion thread.

**Comment by peppy** [comment]
Here's a branch which allows forcing the setting: https://github.com/ppy/osu/compare/master...peppy:osu:nvapi?expand=1

User feedback shows a general preference to having this enabled.

**Reply by peppy** [reply, marked as answer]
This has been resolved via https://github.com/ppy/osu/pull/26354.
